client/src/components/CredentialsButton/CredentialsButton.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./CredentialsButton.css";
+import { PropTypes } from "prop-types";
+
+export const CredentialsButton = ({ data, onClick }) => {
+  return (
+    <button className="credentials" onClick={onClick}>
+      {" "}
+      {data}{" "}
```

Comment:
We should use another variable instead of `data` to give more meaning to what this variable is used. Using data is very generic and at times can be hard to understand

Reply:
Fixed: Changed from `data` to `text`.